instance/models/instance.py

```diff
@@ -586,4 +697,41 @@ def provision(self):
 
         return (server, ansible_log)
 
+    def provision_external_databases(self):
+        """
+        Provision external databases for this instance
+        """
+        self.provision_external_mysql()
+        self.provision_external_mongo()
+
+    def provision_external_mysql(self):
```

Comment:
@omarkhan Would it be worth moving some of these methods to mixin classes? This doesn't seem specific to Open edX instances.